### PR TITLE
User Invite Status Privacy Changes

### PR DESCRIPTION
**Pull request by pxwxnvermx**
[Ticket](https://dimagi.atlassian.net/browse/CCCT-524)

image

### CODE REVIEW

**Review by sravfeyn — APPROVED** [review]

File: commcare_connect/opportunity/tables.py [hunk 1]
```diff
@@ -173,17 +173,24 @@ class Meta:
...
+        if not getattr(record.opportunity_access, "accepted", False):
             return "---"
         return record.opportunity_access.display_name
 
     def render_view_profile(self, record):
-        if record.opportunity_access is None:
-            return "---"
-        if not record.opportunity_access.accepted:
-            return "---"
+        invite_delete_url = reverse(
```
[the remark this answers is not in the document]
Comment (pxwxnvermx) [reply commcare_connect/opportunity/tables.py]: Fixed. [a7cca43](https://github.com/dimagi/commcare-connect/pull/425/commits/a7cca43cb757036d73e63ade7bac0f837a3d5eab)

**Review by calellowitz — COMMENTED** [review]
Does it look weird to have the giant buttons next to the much smaller links?

**Comment by hemant10yadav** [comment]
> Does it look weird to have the giant buttons next to the much smaller links?

> FYI we are going to have another button alongside this delete button for resending invites.

**Comment by pxwxnvermx** [comment]
@calellowitz Changed the button to use the smaller version.

**Review by calellowitz — APPROVED** [review]

**Pull request merged**